doc/config-schema.json

```diff
@@ -98,6 +98,14 @@
 		    "description": "Controller DH-HMAC-CHAP key",
 		    "type": "string"
 		},
+		"keyring": {
+		    "description": "Keyring for TLS key lookup",
+		    "type": "string"
+		},
+		"tls_key": {
+		    "description": "TLS key for the connection",
+		    "type": "string"
+		},
```

Comment:
I think these changes should be in `config-schema.json.in`

Reply:
Of course.